Makefile

```diff
@@ -45,10 +45,9 @@ release-candidate-to-test-pypi: pydist
 	@echo "Release candidate dist built and uploaded to test.pypi.org"
 
 # Build final dist (gets rid of the rc tag) and release final candidate to TestPyPI
-release-final-to-test-pypi: 
+release-final-to-test-pypi:
 	bumpversion --config-file .bumpversion.cfg prerel --allow-dirty --tag
-	pydist
-	pip install twine
```

Comment:
Is twine install no longer needed?

Reply:
it's in the previous command which you have to run before this one so seemed redundant